src/t8_vtk/t8_vtk_writer.hxx

```diff
@@ -256,28 +243,28 @@ class vtk_writer {
     if (curved_flag == 0) {
       switch (element_shape) {
       case T8_ECLASS_VERTEX:
-        pvtkCell = vtkSmartPointer<vtkVertex>::New ();
```

Comment:
If i see it correctly, we could get rid of the complete switch-Block here and replace it with a single
`vecCellIds = new vtkIdType[num_node]`

Reply:
Right